### DISCUSSION TITLE
Lavalink node not connected

### DISCUSSION THREAD

**Discussion by TheDsp21** [SudhanPlayz/Discord-MusicBot#666] [Q&A] [answered]
Hey Bro Sudhan i have a small problem with bot....... when i want to play songs it is telling  Lavalink node not connected can you please help me in this  thankyou...!
![Screenshot (161)](https://user-images.githubusercontent.com/97955637/150384725-fcf07107-33c7-40f1-af9f-25fe633d5958.png)
![Screenshot (161)](https://user-images.githubusercontent.com/97955637/150384751-5100df3b-4d48-4410-9fcd-538483974e61.png)

**Comment by kotru21** [comment, marked as answer]
fill the lavalink field in your botconfig.js![IMG_20220120_210557.jpg](https://user-images.githubusercontent.com/88907641/150396256-28973cf7-0ea8-48f4-8a69-8504a15fb0ad.jpg)
You can find some public lava's here: https://lavalink.darrennathanael.com/
Or host it by yourself

**Reply by gamerzam** [reply]
how do i do it

**Comment by kotru21** [comment]
And, please, next time put the __help-wanted__ label on issues like this(or ask on Discord server if you want).
It is not a bug

**Comment by prayasdev** [comment]
![IMG_20220127_120853](https://user-images.githubusercontent.com/70499403/151305239-0cf580a8-45ea-4b8c-9b5d-5c13b06b8fb0.jpg)
I am having this problem can anyone help me out of this please!!!
Also i set it to secure as i am hosting in repl.it

**Reply by gamerzam** [reply]
me too

**Reply by gamerzam** [reply]
i think we need to create our own lavalink ;-;

**Reply by prayasdev** [reply]
Is the id that is written "main" correct?
Do we have to change this to anything else as we are hosting lavalink on replit?

**Reply by gamerzam** [reply]
ive got a lavalink repl
if u want dm me on discord The Devil Of Clowns#0001
ill give u my lavalink in which my bot works

**Reply by kotru21** [reply]
upd: do not change id main in lava config, you need to change the host, port, password & secure field(if you lava has SSL, for example,replit)

**Comment by kotru21** [comment]
> ![IMG_20220127_120853](https://user-images.githubusercontent.com/70499403/151305239-0cf580a8-45ea-4b8c-9b5d-5c13b06b8fb0.jpg)
> I am having this problem can anyone help me out of this please!!!
> Also i set it to secure as i am hosting in repl.it

Check your config again
If you can,provide more info about your lavalink(and its config)
You may also ask about help in the discord server
